root/static/scripts/tests/GuessCase.js

```diff
@@ -187,6 +187,26 @@ test('Work', function (t) {
             input: "Bring It All To Me (f. 50 Cent)",
             expected: "Bring It All to Me (feat. 50 Cent)",
             mode: "English", roman: true, keepuppercase: false
+        },
+        {
+            input: "izarın gül gül olmuş",
+            expected: "İzarın Gül Gül Olmuş",
+            mode: "Turkish", roman: false, keepuppercase: false
+        },
+        {
+            input: "efendim hu nasibim Bu IECELLİ taksirat yahu",
```

Comment:
Please make _taksirat_ all-uppercase as well so that decomposed character can be tested as well.